packages/block-library/src/archives/index.php

```diff
@@ -79,12 +79,16 @@ function render_block_core_archives( $attributes ) {
 
 		$archives_args['echo'] = 0;
 
-		$block_content = wp_get_archives( $archives_args );
+		$archives = wp_get_archives( $archives_args );
 
-		$block_content = sprintf(
+		$block_content = ! empty( $archives ) ? sprintf(
 			'<ul class="%1$s">%2$s</ul>',
 			esc_attr( $class ),
-			$block_content
+			$archives
+		) : sprintf(
+			'<div class="%1$s">%2$s</div>',
+			$classnames,
```

Comment:
We are outputting an empty class on the frontend `<div class="">No archives to show.</div>`. Should we use `esc_attr( $class )` here instead of `$classnames` ?

Reply:
Thanks for noticing!